### PR TITLE
add orderID to MDC context and log lines

### LINKED ISSUE #1840: add orderID to MDC context and log lines.
- [ ] Add new `ORDER_ID` field to log lines. 
- [ ] Rename TASK_ID to `SERVICE_ID` in the current configuration and code.

### PR DESCRIPTION
**Pull request by baixinsui**
Fixed #1840

### CODE REVIEW

File: modules/deployment/src/main/java/org/eclipse/xpanse/modules/deployment/deployers/terraform/terraformboot/TerraformBootScriptValidator.java [hunk 1]
```diff
@@ -128,4 +123,15 @@ private List<String> getFilesByOcl(Deployment deployment) {
         String deployer = deployment.getDeployer();
         return Collections.singletonList(deployer);
     }
+
+    private UUID getRequestId() {
```
Comment (swaroopar) [inline modules/deployment/src/main/java/org/eclipse/xpanse/modules/deployment/deployers/terraform/terraformboot/TerraformBootScriptValidator.java:127]: i think when we send requests to terraform-boot or tofu-maker the order ID is present. It is better to use that here.

File: modules/models/src/main/java/org/eclipse/xpanse/modules/models/constant/LoggingKeyConstant.java [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+ * SPDX-License-Identifier: Apache-2.0
+ * SPDX-FileCopyrightText: Huawei Inc.
+ */
+
+package org.eclipse.xpanse.modules.models.constant;
+
+/**
+ * Constant keys for logging.
+ */
+public class LoggingKeyConstant {
```
Comment (swaroopar) [inline modules/models/src/main/java/org/eclipse/xpanse/modules/models/constant/LoggingKeyConstant.java:11]: would prefer to keep this class in the logging module.

**Review by swaroopar — CHANGES_REQUESTED** [review]

File: modules/deployment/src/main/java/org/eclipse/xpanse/modules/deployment/deployers/terraform/terraformboot/TerraformBootScriptValidator.java (hunk 1 above)
Comment (baixinsui) [reply modules/deployment/src/main/java/org/eclipse/xpanse/modules/deployment/deployers/terraform/terraformboot/TerraformBootScriptValidator.java:127]: This method is used in register service template, the order id always is null.

File: modules/models/src/main/java/org/eclipse/xpanse/modules/models/constant/LoggingKeyConstant.java (hunk 2 above)
Comment (baixinsui) [reply modules/models/src/main/java/org/eclipse/xpanse/modules/models/constant/LoggingKeyConstant.java]: Moved

**Review by swaroopar — APPROVED** [review]
LGTM

**Pull request merged**